### DISCUSSION TITLE
On Dashboard Account names and the account IDs are not in sync

### DISCUSSION THREAD

**Discussion by Akash-bit-ebg** [aws-solutions-library-samples/cloud-intelligence-dashboards-framework#176] [Q&A]
Hello,

I see that on dashboard, Account names and the account IDs are not in sync if one of those is selected. This means that if you select the account name, the account ID is not automatically selected and vice a versa. Is this issue resolved anywhere?

Thanks,
Akash

**Comment by GL-JesusZambrano** [comment]
Assuming you have the latest setup, edit the Account Map view in Athena to the below:

CREATE OR REPLACE VIEW account_map AS 

/*
Original Code, edited to resolve incorrect mapping configuration
SELECT DISTINCT
  "line_item_usage_account_id" "account_id"
, "line_item_usage_account_id" "account_name"
FROM
  cudos_dashboard_cur
*/

SELECT DISTINCT
  "account_id"
, "account_name"
FROM
  aws_accounts 

Also check the code being used by the aws_account view and make sure the data is correctly configured.
